src/include/miopen/handle.hpp

```diff
@@ -51,10 +51,14 @@
 #include <unordered_map>
 
 #if MIOPEN_USE_ROCBLAS
+#if defined(_WIN32) && (HIP_PACKAGE_VERSION_FLAT < 5007000000ULL)
+#define ROCBLAS_BETA_FEATURES_API 0
+#else
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-macros"
 #define ROCBLAS_BETA_FEATURES_API 1
 #pragma clang diagnostic pop
+#endif
```

Comment:
[Side note] @JehandadKhan BTW is seems like handle.hpp does not use any beta API features from rocblas. Shall we simply remove ROCBLAS_BETA_FEATURES_API here?

Reply:
Yes, please

Reply:
@JehandadKhan Oh, no. We have to define ROCBLAS_BETA_FEATURES_API in each header where rocblas.h is included. Because the order of inclusion of headers is not guaranteed, and rocblas.h in included only once, at the first appearance!

Reply:
@JehandadKhan ...and, as genn_v2.cpp includes handle.hpp, removal of ROCBLAS_BETA_FEATURES_API from handle.hpp would lead to that rocblas.h is included without beta stuff (from handle.hpp).

Reply:
@JehandadKhan 
> We have to define ROCBLAS_BETA_FEATURES_API in each header where rocblas.h is included

I've quickly inspected the code and found that rocblas.h is included only in handle.hpp and gemm_v2.cpp, so we are fine.

[end of the side thread]